app/wiring/AppComponents.scala

```diff
@@ -29,26 +29,28 @@ class AppComponents(context: Context, stage: String) extends BuiltInComponentsFr
 
   // https://github.com/guardian/play-googleauth#implement-googlegroups-based-access-control-using-the-directory-api
   private val googleGroupChecker = {
-    val s3URI = new AmazonS3URI(configuration.get[String]("googleAuth.serviceAccount.certificateS3Path"))
```

Comment:
before it was fetching from SSM the bucket/key of the credentials json file. This is a bit pointless as the bucket is referenced elsewhere in this repo.